src/py21cmfast/src/SpinTemperatureBox.c

```diff
@@ -1864,6 +1969,16 @@ LOG_SUPER_DEBUG("looping over box...");
                                                                     freq_int_lya_tbl[m_xHII_low_box[box_ct]][R_ct] ));
                             dstarlya_dt_box[box_ct] += (double)del_fcoll_Rct[box_ct]*dstarlya_dt_prefactor[R_ct];
 
+                            //Add Lya flux
+                            if (flag_options->USE_Lya_HEATING){
+                                dstarlya_cont_dt_box[box_ct] += (double)del_fcoll_Rct[box_ct]*dstarlya_cont_dt_prefactor[R_ct];
+                                dstarlya_inj_dt_box[box_ct] += (double)del_fcoll_Rct[box_ct]*dstarlya_inj_dt_prefactor[R_ct];
+                                if (flag_options->USE_MINI_HALOS){
```

Comment:
This part should be moved a few line below into the section of 
if (flag_options->USE_MINI_HALOS){
       if(ave_fcoll_MINI!=0.) {

Reply:
Okay.